sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2588,6 +2588,9 @@ public void deleteWithContacts(String caseUuid, DeletionDetails deletionDetails)
 	private void deleteCase(Case caze, DeletionDetails deletionDetails)
 		throws ExternalSurveillanceToolRuntimeException, SormasToSormasRuntimeException {
 
+		if (!caseService.inJurisdictionOrOwned(caze)) {
```

Comment:
If you have time, it would be great to have some unit tests for deleteCase, deleteCases in CaseFacadeEjbTest
and deleteContact, deleteContacts in ContactFacadeEjbTest with this scenario ( when the jurisdiction is different and when is the same and these methods are called).

Reply:
tests added